docs/_running_external.rst

```diff
@@ -0,0 +1,223 @@
+A list of all created profiles is provided by the ``profile_list`` subcommand:
+
+::
+
+    $ planemo profile_list
+    Looking for profiles...
+    tutorial_profile
+    usegalaxy-eu
+    usegalaxy-org
+    3 configured profiles are available.
+
+
+Once the new ``tutorial_profile`` is created, a workflow can be executed with:
+
+::
+
+    $ planemo run wf3-shed-tools.ga wf3-shed-tools-job.yml --profile tutorial_profile
+
+
+Generating the job file
+===============================================
+
+The example workflow used so far provides not only the workflow, but also the
+job file which specifies the inputs to be used. If you have created and downloaded
+your own workflow, you need to create this job file yourself. Planemo provides a
+handy command which provides a template for this:
+
+::
+
+    $ planemo workflow_job_init wf3-shed-tools.ga -o test-job-file.yml
+
+
+Opening ``test-job-file.yml`` should show the following:
+
+::
+
+    GSE37268_mof3:
+      class: File
+      path: todo_test_data_path.ext
+    Genes:
+      class: File
+      path: todo_test_data_path.ext
+
+
+For each of the specified inputs in the workflow, a entry is created in the
+output YAML file. In this case, both inputs are datasets rather than collections
+or parameters, so they are both classified as ``class: File``. A placeholder
+path is also included, which you should change to the paths of your chosen
+input files.
+
+Another more complex example, also including a collection and parameter as
+input, might look like the following:
+
+::
+
+    input_dataset:
+      class: File
+      path: todo_test_data_path.ext
+    input_collection:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: todo_element_name
+        path: todo_test_data_path.ext
+    input_parameter: param_todo
+
+
+For the collection, each dataset is listed, with a path and identifier specified.
+For the parameter, only a single value has to be specified, but ensure it is of
+the correct type, e.g. float, if that is what the workflow requires. 
+
+Using workflow and dataset IDs
+===============================================
+
+If you ran all the commands above then you probably noticed that both the
+workflow and the input datasets get newly uploaded at each execution. If you
+want to run the same workflow multiple times, you may prefer to avoid this.
+In the examples given so far, all workflows and datasets are specified by means
+of a local path, but Planemo also allows you to use the IDs created by Galaxy
+as well. These IDs are unique to each Galaxy server, so this approach isn't
+transferrable if you want to run your workflows on multiple servers.
+
+The first step is to ensure all the datasets which are required for the
+workflow are already uploaded. You can either do this by running the workflow
+once in the normal way, as described above, or just manually uploading through
```

Comment:
Can you please create a planemo issue about this one? I guess a planemo upload_data command might be useful that returns the IDs?